litmus-portal/frontend/src/pages/HomePage/index.tsx

```diff
@@ -182,36 +182,40 @@ const HomePage: React.FC = () => {
                   </div>
                 </div>
               )}
-              <div className={classes.contentDiv}>
-                <div className={classes.statDiv}>
-                  <div className={classes.btnHeaderDiv}>
-                    <Typography className={classes.statsHeading}>
-                      <strong>{t('home.analytics.heading')}</strong>
-                    </Typography>
+              {!dataPresent ? (
+                <div className={classes.contentDiv}>
+                  <div className={classes.statDiv}>
+                    <div className={classes.btnHeaderDiv}>
+                      <Typography className={classes.statsHeading}>
+                        <strong>{t('home.analytics.heading')}</strong>
+                      </Typography>
 
-                    <Button
-                      className={classes.seeAllBtn}
-                      onClick={(event) => {
-                        event.preventDefault();
-                        history.push('/community');
-                      }}
-                    >
-                      <div className={classes.btnSpan}>
-                        <Typography className={classes.btnText}>
-                          {t('home.analytics.moreInfo')}
-                        </Typography>
-                        <img src="icons/next.png" alt="next" />
-                      </div>
-                    </Button>
+                      <Button
```

Comment:
Please will use the present Button component?

Reply:
It was already there @oumkale

Reply:
I haven't made changes to existing components used, this can go in as separate bug-fix/enhancement PR.

Reply:
can you fix this in this PR itself @ishangupta-ds ?

Reply:
The styles are breaking @arkajyotiMukherjee on using that, also I am having trouble with the analytics data in place, that component is not getting displayed half of the times. This we can fix with another PR, much style changes are required there..